summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormTopologyBuilder.scala

```diff
@@ -1,102 +1,262 @@
 package com.twitter.summingbird.storm
 
-import com.twitter.summingbird.Options
-import com.twitter.summingbird.batch.{ BatchID, Timestamp }
+import com.twitter.algebird.Semigroup
+import com.twitter.summingbird.{ Options, Summer }
+import com.twitter.summingbird.batch.{ BatchID, Batcher, Timestamp }
 import com.twitter.summingbird.online.executor
+import com.twitter.summingbird.online.executor.KeyValueShards
 import com.twitter.summingbird.option.JobId
 import com.twitter.summingbird.planner.{ Dag, FlatMapNode, SourceNode, SummerNode }
 import com.twitter.summingbird.storm.Constants._
-import com.twitter.summingbird.storm.builder.Topology.ReceivingId
-import com.twitter.summingbird.storm.builder.{ EdgeType, Topology }
+import com.twitter.summingbird.storm.builder.Topology
 import com.twitter.summingbird.storm.planner.StormNode
 import org.apache.storm.generated.StormTopology
 import org.slf4j.LoggerFactory
 import scala.reflect.ClassTag
-
 import scala.collection.{ Map => CMap }
 
-object StormTopologyBuilder {
+/**
+ * This class contains main logic on how to build storm topology out of planned DAG.
+ * `DAG` contains three types of nodes: Source, FlatMap and Summer.
```

Comment:
I changed to [[]], but tbh I dislike it =( I think backticks much lighter and easier to read. But I don't mind to use [[]].